lib/skroutz.rb

```diff
@@ -16,6 +16,7 @@ module Skroutz
   autoload :Default, 'skroutz/default'
   autoload :Parsing, 'skroutz/parsing'
   autoload :Resource, 'skroutz/resource'
+  autoload :Associations, 'skroutz/associations'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.